src/IW/Core/PGArray'.hs

```diff
@@ -0,0 +1,22 @@
+{-# LANGUAGE DeriveAnyClass #-}
+
+module IW.Core.PGArray'
+    ( PGArray' (..)
+    ) where
+
+import Database.PostgreSQL.Simple.Types (PGArray (..))
+import Database.PostgreSQL.Simple.FromField (FieldParser, pgArrayFieldParser)
+import Database.PostgreSQL.Simple.ToField (Action (..))
+
+
+newtype PGArray' a = PGArray' { fromPGArray' :: [a] }
```

Comment:
After you rename it to `SqlArray` the name of the field should be `unSqlArray` according to our style guide.

Reply:
Got it.